health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponentView.kt

```diff
@@ -140,6 +142,12 @@ class PaymentComponentView(context: Context, attrs: AttributeSet?) : ConstraintL
         }
     }
 
+    private fun checkPaymentComponentHeight() {
+        if (resources.getDimension(R.dimen.ghs_payment_component_height) > resources.getDimension(R.dimen.ghs_accessibility_min_height)) {
```

Comment:
[Request] We should use `>=` so that the height can be set to a height equal to the minimum.

Reply:
Yes, an oversight on my part - applied this change.